drush/Commands/DeployCommands.php

```diff
@@ -378,10 +386,13 @@ public function deploy(string $target, string $git_ref, array $options = []) {
     $done = $this->getTimestamp();
     $this->io()->success("Deployment completed at {$done}");
 
-    // Process purge queue.
-    $process = Drush::drush($targetRecord, 'p:queue-work', [], ['finish' => TRUE, 'verbose' => TRUE]);
-    $process->mustRun();
-    $this->logger()->success("Purge queue worker complete at $target.");
+
+    if (!$options['varnish']) {
+      // Process purge queue.
+      $process = Drush::drush($targetRecord, 'p:queue-work', [], ['finish' => TRUE, 'verbose' => TRUE]);
+      $process->mustRun();
+      $this->logger()->success("Purge queue worker complete at $target.");
```

Comment:
```suggestion
      $this->logger()->success("Purge queue worker complete at $targetRecord.");
```

Reply:
Hmmm. $targetRecord is supposed to be a SiteAlias object, and $target is a string. Are you seeing differently?

Reply:
Ah okay. This is likely the wrong fix then. What I was seeing was that the target was blank in the log output.

Reply:
Fix submitted

Reply:
Thanks! Looks good to me